src/kOS.Safe/Encapsulation/ISerializableValue.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    public interface ISerializableValue
```

Comment:
I'm using that interface to basically group all new encapsulation types as they had no common parent class that I could use to recognize them. I'm not familiar with C# attributes, so I don't know if we could use them to replace that.